src/modules/MouseUtils/InclusiveMouse/InclusiveCrosshair.cpp

```diff
@@ -0,0 +1,452 @@
+    ptMonitorBottomRight.y = monitorInfo.rcMonitor.bottom;
+
+    // Convert everything to client coordinates.
+    ScreenToClient(m_hwnd, &ptCursor);
+    ScreenToClient(m_hwnd, &ptMonitorUpperLeft);
+    ScreenToClient(m_hwnd, &ptMonitorBottomRight);
+
+    // Position crosshair components around the mouse pointer.
+    m_left_crosshair_border.Offset({ (float)ptCursor.x - m_crosshair_radius + m_crosshair_border_size, (float)ptCursor.y, .0f });
+    m_left_crosshair_border.Size({ (float)ptCursor.x - (float)ptMonitorUpperLeft.x - m_crosshair_radius + m_crosshair_border_size, m_crosshair_thickness + m_crosshair_border_size * 2 });
+    m_left_crosshair.Offset({ (float)ptCursor.x - m_crosshair_radius, (float)ptCursor.y, .0f });
+    m_left_crosshair.Size({ (float)ptCursor.x - (float)ptMonitorUpperLeft.x - m_crosshair_radius, m_crosshair_thickness });
+
+    m_right_crosshair_border.Offset({ (float)ptCursor.x + m_crosshair_radius - m_crosshair_border_size, (float)ptCursor.y, .0f });
+    m_right_crosshair_border.Size({ (float)ptMonitorBottomRight.x - (float)ptCursor.x - m_crosshair_radius + m_crosshair_border_size, m_crosshair_thickness + m_crosshair_border_size * 2 });
+    m_right_crosshair.Offset({ (float)ptCursor.x + m_crosshair_radius, (float)ptCursor.y, .0f });
+    m_right_crosshair.Size({ (float)ptMonitorBottomRight.x - (float)ptCursor.x - m_crosshair_radius, m_crosshair_thickness });
+
+    m_top_crosshair_border.Offset({ (float)ptCursor.x, (float)ptCursor.y - m_crosshair_radius + m_crosshair_border_size, .0f });
+    m_top_crosshair_border.Size({ m_crosshair_thickness + m_crosshair_border_size * 2, (float)ptCursor.y - (float)ptMonitorUpperLeft.y - m_crosshair_radius + m_crosshair_border_size });
+    m_top_crosshair.Offset({ (float)ptCursor.x, (float)ptCursor.y - m_crosshair_radius, .0f });
+    m_top_crosshair.Size({ m_crosshair_thickness, (float)ptCursor.y - (float)ptMonitorUpperLeft.y - m_crosshair_radius });
+
+    m_bottom_crosshair_border.Offset({ (float)ptCursor.x, (float)ptCursor.y + m_crosshair_radius - m_crosshair_border_size, .0f });
+    m_bottom_crosshair_border.Size({ m_crosshair_thickness + m_crosshair_border_size * 2, (float)ptMonitorBottomRight.y - (float)ptCursor.y - m_crosshair_radius + m_crosshair_border_size });
+    m_bottom_crosshair.Offset({ (float)ptCursor.x, (float)ptCursor.y + m_crosshair_radius, .0f });
+    m_bottom_crosshair.Size({ m_crosshair_thickness, (float)ptMonitorBottomRight.y - (float)ptCursor.y - m_crosshair_radius });
+
+}
+
+LRESULT CALLBACK InclusiveCrosshair::MouseHookProc(int nCode, WPARAM wParam, LPARAM lParam) noexcept
+{
+    if (nCode >= 0)
+    {
+        MSLLHOOKSTRUCT* hookData = (MSLLHOOKSTRUCT*)lParam;
+        if (wParam == WM_MOUSEMOVE) {
+            instance->UpdateCrosshairPosition();
+        }
+    }
+    return CallNextHookEx(0, nCode, wParam, lParam);
+}
+
+void InclusiveCrosshair::StartDrawing()
+{
+    Logger::info("Start drawing crosshairs.");
```

Comment:
Small tip: you can use `CallTracer` here instead of writing it manually.
https://github.com/microsoft/PowerToys/blob/main/src/common/logger/call_tracer.h

Reply:
Cool tip for the future. Thank you!
I do prefer it written like this, though :)
It's also info rather than trace here.